portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,12 +31,19 @@
 /** Servlet that returns comment data as a JSON list */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  // Create an ArrayList to store the comments 
-  private static final ArrayList<String> commentsList = new ArrayList<String>();
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Once you resolve the comment [1], you should be able to reuse those `private static final String` variables.

[1] https://github.com/anthonyqzhu/step/pull/13#discussion_r434889178